rust/src/connection/network/transmitter/rpc.rs

```diff
@@ -151,7 +153,28 @@ impl RPCTransmitter {
             Request::Transaction(transaction_request) => {
                 let req = transaction_request.into_proto();
                 let req_id = RequestID::from(req.req_id.clone());
-                let (request_sink, response_source) = rpc.transaction(req).await?;
+                let (request_sink, mut response_source) = rpc.transaction(req).await?;
+                match response_source.next().await {
+                    Some(Ok(transaction::Server { server: Some(Server::Res(res)) })) => {
+                        match TransactionResponse::try_from_proto(res) {
+                            Ok(TransactionResponse::Open { server_duration_millis }) => {
+                                // let open_latency =
```

Comment:
Will implement it in a half an hour... Does not seem used now though, can be left as a todo?

Reply:
we should still be figuring out the latency update I think, can you leave a comment `// TODO ` here?

Reply:
We can return it as a part of `Response::TransactionStream`

Reply:
@dmitrii-ubskii Good point! Thanks, will do on Monday. Will leave a todo